src/client/sandbox/native-methods.js

```diff
@@ -310,6 +312,13 @@ class NativeMethods {
         this.elementInnerHTMLSetter     = elementInnerHTMLDescriptor.set;
         this.elementOuterHTMLSetter     = elementOuterHTMLDescriptor.set;
 
+        // NOTE: Some browsers (for example, Edge, Internet Explorer 11, Safari) have no 'integrity' property
```

Comment:
// NOTE: Some browsers (for example, Edge, Internet Explorer 11, Safari) do not support an `integrity` attribute.